news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,75 @@
+
+The community was trying their best to hunt for the solution, but after an enormous number of attempts that they thought would surely work, insanity started creeping in, and hope was slowly starting to fade. 
+
+But suddenly, after more than 3 weeks, the very last hint was dropped by [Repflez](https://osu.ppy.sh/users/201392) saying that "a message in the last 8 hours was a hint in the right direction", and after reading it, UberFazz found that message and figured exactly what they needed to do. That theory was to set scores with particular combos that relate to the RGB values of a difficulty, which UberFazz guessed was on the "celinsane" difficulty as it literally meant cell + insane just like the medal suggested. Shortly after trying this idea in regards to combo matching each value of the RGB, UberFazz successfully got **Internment** medal and received the "**Medal Hunter**" title, after more than a month since the medal's release!
+
+![](/wiki/shared/news/2023-08-28-medal-hunter-titles-second-coming/uberfazz_profile.png)
+
+After giving it more tests which crossed out grades from the equation and being celinsane diff specific, the community was finally ready to present the full solution to **Internment** medal:
+
+On any "Insane" difficulty (4.0\* - 5.29\*) of [Frums - theyaremanycolors](https://osu.ppy.sh/beatmapsets/1416051#osu/3022086), set three plays in a row that have combos equal to the R, G, and B values of the difficulty indicator (on the osu! website) for the map you are playing. The possible combinations are as follows:
+
+| Play | Combo ([vanchanicalinsane](https://osu.ppy.sh/beatmapsets/1416051#osu/3022086)) | Combo ([celinsane](https://osu.ppy.sh/beatmapsets/1533352#osu/4052455)) |
+| --- | --- | --- |
+| 1st | 255x | 243x |
+| 2nd | 104x | 76x |
+| 3rd | 108x | 133x |
+
+You also MUST get the scores back-to-back, and the third one must be a pass; If you overcombo at any point on the second or third play, fail the third play, or fail to reach the max combo requirement for the second or third score, you must restart the medal from the beginning.
+
+And now, after it's found, UberFazz would like to give massive thanks to all those that participated in this event. They say that without the osu! medal hunting community, this would have never transpired. There were very incredibly close guesses that barely undershot the medal solution, but UberFazz was the one that made that final push and emerged as the ultimate victor for Internment and its medal hunter title.
```

Comment:
```suggestion
And now, after this great struggle, UberFazz would like to give massive thanks to all those that participated in this event. They say that without the osu! medal hunting community, this would have never transpired. There were very incredibly close guesses that barely undershot the medal solution, but UberFazz was ultimately the one that made that final push and emerged as the ultimate victor.
```